daft/expressions2/expressions.py

```diff
@@ -172,6 +172,10 @@ def cast(self, dtype: DataType) -> Expression:
         expr = self._expr.cast(dtype._dtype)
         return Expression._from_pyexpr(expr)
 
+    def sum(self) -> Expression:
```

Comment:
Lets put this as `_sum` since we don't want to expose this quite yet as top level. I believe we do this now